src/utils/common.ts

```diff
@@ -5,7 +5,7 @@
  * @returns Parameter string with array brackets [] removed.
  */
 export const stripArrayTypeIfPresent = (typeString: string) => {
-  if (typeString?.match(/\S\[\d*\]$/u) !== null) {
+  if (typeString?.match(/\S\[\d*\]$/u)) {
```

Comment:
I found this condition not working as expected when `typeString` was `undefined`.

Reply:
Makes sense! Could you add a test case for this?

Reply:
PR is updated with test case.